lib/mix/lib/mix/tasks/format.ex

```diff
@@ -65,6 +71,43 @@ defmodule Mix.Tasks.Format do
   of patterns and files to `mix format`, as showed at the top of this task
   documentation. This list can also be set in the `.formatter.exs` under the
   `:inputs` key.
+
+  ## Importing dependencies configuration
+
+  This task supports importing formatter configuration from dependencies.
+
+  A dependency that wants to export formatter configuration needs to have a `.formatter.exs` file
```

Comment:
strictly speaking, for Hex dependencies the `.formatter.exs` file needs to be in the package and by default it won't be [1]. I think we could add it by default, or make a note here, or hope maintainers will remember to set it. WDYT?

[1] https://github.com/hexpm/hex/blob/v0.17.1/lib/mix/tasks/hex.build.ex#L4

Reply:
@wojtekmach let's update hex?